src/ebay-fake-menu/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react';
+import initStoryshots from '@storybook/addon-storyshots'
+import { EbayFakeMenu, EbayFakeMenuItem } from '../index'
+
+initStoryshots({
+    config: ({ configure }) => configure(() => require('./index.stories'), module)
+})
+
+describe('<EbayFakeMenu>', () => {
+    describe('on menu item click', () => {
+        it('should fire onClick event', () => {
+            const onClickSpy = jest.fn()
+            const wrapper = render(
+                <EbayFakeMenu>
+                    <EbayFakeMenuItem href="#" onClick={onClickSpy}/>
+                </EbayFakeMenu>
+            )
+
+            fireEvent.click(wrapper.container.querySelectorAll('.fake-menu__item')[0])
```

Comment:
```suggestion
            fireEvent.click(wrapper.container.querySelectorAll('.fake-menu__item')[0])

            expect(onClickSpy).toBeCalled()            
```

You forgot the assertion